buildpacks/nodejs-npm-install/tests/snapshots/test_prune_dev_dependencies_config.snap

```diff
@@ -0,0 +1,40 @@
+---
+source: test_support/src/lib.rs
+---
+## Heroku Node.js Engine
+
+- Checking Node.js version
+  - Detected Node.js version range: `>=22.0.0 <23.0.0-0`
+  - Resolved Node.js version: `22.17.0`
+- Installing Node.js distribution
+  - GET https://nodejs.org/download/release/v22.17.0/node-v22.17.0-<arch>.tar.gz ... (OK)
+  - Downloading ... (<time_elapsed>)
+  - Verifying checksum
+  - Extracting Node.js `22.17.0 (<arch>)`
+  - Installing Node.js `22.17.0 (<arch>)` ... (<time_elapsed>)
+- Done (finished in <time_elapsed>)
+
+## Heroku Node.js npm Install
+
+- Installing node modules
+  - Using npm version `10.9.2`
+  - Creating npm cache
+  - Configuring npm cache directory
+  - Running `npm ci`
+
+
+      added <NUMBER> packages, and audited <NUMBER> packages in <time_elapsed>
+
+      found 0 vulnerabilities
+
+  - Done (<time_elapsed>)
+- Running scripts
+  - No build scripts found
+- Pruning dev dependencies
+  - Skipping as pruning was disabled in project.toml
```

Comment:
Suggestion (not blocking): I think we should tell people specifics of where the warning is coming from. Either list the key and file here or earlier so we don't have to make them guess (even if it's fairly obvious now, lets set a positive pattern so if we add a lot more keys in the future they won't have to guess).